agent/tool-scripts/postprocess/samples/prometheus-metrics/metrics-stdout

```diff
@@ -0,0 +1,380 @@
+{"metrics": [{"value": "22"}], "@timestamp": 1491342293, "type": "COUNTER", "name": "admission_quota_controller_adds"}
```

Comment:
Is there any difference between the `gold/prometheus-metrics/metrics-stdout` and this file?  If not, can we eliminate the `gold` one somehow?